dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/RumEventSerializer.kt

```diff
@@ -118,6 +143,10 @@ internal class RumEventSerializer : Serializer<RumEvent> {
         internal const val TAG_ENV = "env"
         internal const val TAG_SERVICE = "service"
         internal const val TAG_MESSAGE = "message"
+        internal const val TAG_USER = "user"
+        internal const val TAG_USER_ID = "id"
+        internal const val TAG_USER_EMAIL = "email"
+        internal const val TAG_USER_NAME = "name"
```

Comment:
You know, you don't have to create a user object if you simply use `user.id`, `user.email` directly

Reply:
even better ...tks